serverlessworkflow/modules/ROOT/pages/_common-content/camel-valid-responses.adoc

```diff
@@ -0,0 +1,5 @@
+:page-partial:
+1. A valid JSON string
```

Comment:
This is confusing
A JSON string might be understoon as a string data type
So I would write, a String that contains a json complex object